package.json

```diff
@@ -25,43 +25,43 @@
     "lit-html": "^1.4.1"
   },
   "peerDependencies": {
-    "@alaskaairux/auro-icon": "^3.2.0",
-    "@alaskaairux/design-tokens": "^3.4.0",
-    "@alaskaairux/icons": "^4.15.0",
-    "@alaskaairux/webcorestylesheets": "^3.7.2",
+    "@alaskaairux/auro-icon": "^3.4.0",
+    "@alaskaairux/design-tokens": "^3.12.2",
+    "@alaskaairux/icons": "^4.20.2",
+    "@aurodesignsystem/webcorestylesheets": "^4.3.0",
```

Comment:
I am guessing that this was a manual change to the package.json? Project is unable to build due to incorrect package-lock resource. 

https://github.com/AlaskaAirlines/auro-input/actions/runs/4474198121/jobs/7862404994#step:4:20